### PR TITLE
Feat: Adds tokens to `ImageGallery`

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

This PR applies the new https://github.com/vtex-sites/base.store/pull/407 to `ImageGallery`: new scoped tokens linked to global ones.

image

## How does it work?

This PR uses local variables to parameterize the `ImageGallery` properties, then connects these scoped tokens to the global ones.

## How to test it?

- The `ImageGallery` component and its related components should look just as it was before these changes. [Preview Link](https://sfj-448c3b0--nextjs.preview.vtex.app/apple-magic-mouse-99988212/p)
- You can check the story on [Storybook Preview](https://nextjs-store-storybook-git-theming-image-galle-14cdde-faststore.vercel.app/?path=/docs/organisms-imagegallery--overview-default).

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [x] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

File: src/components/ui/ImageGallery/image-gallery-selector.module.scss [hunk 1]
```diff
@@ -1,13 +1,89 @@
...
+  --fs-image-gallery-selector-max-height                        : 33.125rem;   // 530px
+
+  // Elements
+  --fs-image-gallery-selector-elements-gap                      : var(--fs-spacing-1);
+  --fs-image-gallery-selector-elements-gap-notebook             : var(--fs-spacing-2);
+  --fs-image-gallery-selector-elements-padding                  : var(--fs-spacing-0);
+
+  // Thumbnail
+  --fs-image-gallery-selector-thumbnail-width-mobile            : 3.5rem;   // 56px
+  --fs-image-gallery-selector-thumbnail-width-notebook          : 4.5rem;   // 72px
```
Comment (lucasfp13) [inline src/components/ui/ImageGallery/image-gallery-selector.module.scss:18]: Is there some reason to not use global spacing tokens here? (e.g. `--fs-spacing-8` and `--fs-spacing-10`)

**Review by lucasfp13 — APPROVED** [review]
LGTM!

Just left some comments, nice job ✨

**Review by renatamottam — APPROVED** [review]
Nice one! 👏

File: src/components/ui/ImageGallery/image-gallery-selector.module.scss [hunk 2]
```diff
@@ -1,13 +1,89 @@
...
+  --fs-image-gallery-selector-thumbnail-border-radius           : var(--fs-border-radius);
+  --fs-image-gallery-selector-thumbnail-border-color-selected   : var(--fs-border-color-active);
```
[the remark this answers is not in the document]
Comment (hellofanny) [reply src/components/ui/ImageGallery/image-gallery-selector.module.scss:20]: Checked! That's right! I'll update it :) Thanksss

**Pull request merged**